src/Catalyst.Cli/Commands/GetDeltaCommand.cs

```diff
@@ -46,7 +45,7 @@ protected override GetDeltaRequest GetMessage(GetDeltaOptions option)
             {
                 var hashBytes = MultiBase.Decode(option.Hash);
                 var cid = hashBytes.ToCid();
-                var multiHash = new MultiHash(cid.Hash.ToArray());
+                new MultiHash(cid.Hash.ToArray());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'MultiHash' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)